_news/2018-06-07-forseti-2.0-launch.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Forseti 2.0 Launch
+author: Henry Chang
+---
+Forseti 1.0 was initially launched at Google Cloud Next '17 a little over a year ago. Since then, Forseti went on to be adopted by over 100 businesses and organizations, helping to secure Google Cloud Platform environments large and small.
+
+We received great feedback on how Forseti could be improved, as well as insight into additional features you like to see, which require making fundamental architectural changes to Forseti in order to make the implementation possible.  The Forseti team embarked on a journey to make these changes, in order to take Forseti to the next level.  Thanks to hard work with many contributors, we are excited to share with you some of the exciting new changes in Forseti 2.0 below!
```

Comment:
Overall comments:
- Please keep your character line length to 80, if that causes wrapping on a URL that's fine.
- One space after a period